### PR TITLE
Update theme.js

### PR DESCRIPTION
**Pull request by ankur0007**
Ticket - https://core.trac.wordpress.org/ticket/61578

### CODE REVIEW

File: src/js/_enqueues/wp/theme.js [hunk 1]
```diff
@@ -118,9 +119,11 @@ themes.view.Appearance = wp.Backbone.View.extend({
 		// Render and append after screen title.
 		view.render();
 		this.searchContainer
+			.find('.search-box')
 			.append( $.parseHTML( '<label for="wp-filter-search-input">' + l10n.search + '</label>' ) )
```
Comment (Debarghya-Banerjee) [inline src/js/_enqueues/wp/theme.js]:
```suggestion
.find( '.search-box' )
.append( $.parseHTML( `<label for="wp-filter-search-input">${ 10n.search }</label>` ) )
```

**Review by Debarghya-Banerjee — CHANGES_REQUESTED** [review]

File: src/js/_enqueues/wp/theme.js [hunk 2]
```diff
@@ -96,7 +96,8 @@ themes.view.Appearance = wp.Backbone.View.extend({
 	},
 
 	// Defines search element container.
-	searchContainer: $( '.search-form .search-box' ),
+	//searchContainer: $( '.search-form .search-box' ),//Bug => //Submit only works with Form and not with <p> tag
+	searchContainer: $( '.search-form' ),//Fix
```
Comment (sabernhardt) [inline src/js/_enqueues/wp/theme.js]:
```suggestion
	searchContainer: $( '.search-form' ),
```

remove comments

File: src/js/_enqueues/wp/theme.js [hunk 3]
```diff
@@ -118,9 +119,11 @@ themes.view.Appearance = wp.Backbone.View.extend({
...
 		this.searchContainer
+			.find('.search-box')
 			.append( $.parseHTML( '<label for="wp-filter-search-input">' + l10n.search + '</label>' ) )
-			.append( view.el )
-			.on( 'submit', function( event ) {
+			.append( view.el );
+
+		this.searchContainer.on( 'submit', function( event ) {
 				event.preventDefault();
 			});
```
Comment (sabernhardt) [inline src/js/_enqueues/wp/theme.js]:
```suggestion
			event.preventDefault();
		});
```

These two lines do not need as much indentation.

File: src/js/_enqueues/wp/theme.js [hunk 4]
```diff
@@ -118,9 +119,11 @@ themes.view.Appearance = wp.Backbone.View.extend({
 		// Render and append after screen title.
 		view.render();
 		this.searchContainer
+			.find('.search-box')
```
Comment (sabernhardt) [inline src/js/_enqueues/wp/theme.js]:
```suggestion
			.find( '.search-box' )
```

add spaces

File: src/js/_enqueues/wp/theme.js (hunk 1 above)
Comment (sabernhardt) [reply src/js/_enqueues/wp/theme.js]: The `'.search-box'` selector could have spaces around it (as noted above), but these lines need to [remain indented](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#chained-method-calls).

**Review by sabernhardt — COMMENTED** [review]
This worked for me in testing, but you could remove the comments and edit some spacing and indentation.

**Comment by ankur0007** [comment]
All suggested changes are applied, thank you :)

**Comment by jeherve** [comment]
This tests well for me on 6.6.1. Thanks for working on a fix!

**Comment by ankur0007** [comment]
Awesome, Thank you!

**Review by hellofromtonya — APPROVED** [review]
Tested for issues with [Appearance > Themes](https://core.trac.wordpress.org/ticket/61578#comment:21) and [Appearance > Themes > Add Theme](https://core.trac.wordpress.org/ticket/61578#comment:20).

✅  This patch resolves the issues with both search workflows.

LGTM for commit.

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/58914.

**Pull request closed without merging**